sphinx_rtd_theme/__init__.py

```diff
@@ -31,6 +31,12 @@ def config_initiated(app, config):
             _('The canonical_url option is deprecated, use the html_baseurl option from Sphinx instead.')
         )
 
+
+def extend_html_context(app, pagename, templatename, context, doctree):
+     # Add ``sphinx_version_info`` tuple for use in Jinja templates
+     context['sphinx_version_info'] = sphinx_version
```

Comment:
Note: ``sphinx_version`` is now just the documented value at https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.version_info.

I could definitely imagine regretting only having the first 3 elements from that object in the future, and it makes no difference to the existing usage of the ``sphinx_version_info`` variable (the tuple comparison works the same if it is just the 3 elements, or the full thing)

Reply:
So just to get you correctly: `sphinx_version_info` is now completely identical to `sphinx_version` and is just an alias because we cannot know if `sphinx_version_info` was used by other themes or template customization, right?

Reply:
I'm mostly certain we'll never need more than major and minor versions here, but the additional specifiers don't seem like they'll hurt anything either. This seems fine, unless we hit something with downstream usage.